### DISCUSSION TITLE
fbx로 임포팅한 애니메이션 파일을 재생할 수 있는 방법이 있을까요?

### DISCUSSION THREAD

**Discussion by deanvilgel** [naverz/zepeto-studio-kor#293] [5. Q&A]
코드로 작성하는 방법말고, fbx로 가져온 애니메이션을 제페토 내에서도 재생하는 방법이 있는지 알고 싶습니다!
아바타애니메이션이 아닌 월드의 물체들에 애니메이션을 적용하고 싶습니다

**Comment by x4cf** [comment]
이런 애니메이션일까요?
https://github.com/naverz/zepeto-studio-kor/discussions/163

**Reply by deanvilgel** [reply]
제가 원하는 건 물체 등이 움직이는 거지만, 이것도 원리는 비슷할 거 같아요! 근데 이런 애니메이션도 모바일 환경 제페토에서도 적용할 수 있나요? 제가 해볼 때는 zepetoscript로 작성한게 아닌 fbx에서 가져온 애니메이션은 적용되지 않아서요 ㅠ

**Reply by x4cf** [reply]
fbx로 가져온 애니메이션도 잘 적용되요 :D
